src/main/java/org/broadinstitute/hellbender/tools/coveragemodel/TargetLikelihoodCalculator.java

```diff
@@ -0,0 +1,15 @@
+package org.broadinstitute.hellbender.tools.coveragemodel;
+
+import org.broadinstitute.hellbender.tools.exome.Target;
+
+/**
+ * Calculate log-likelihoods of observed coverage as a function of copy ratio.  This is needed for likelihood-based segmentation
+ * and calling of CNVs, for example our HMM for germline CNVs.
```

Comment:
Maybe a little more discussion of what constitutes a "coverage" here in the doc?  Is any double associated with a target OK?